openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -42,7 +42,7 @@ def _directory_name(data_usage_key):
     )
 
 
-def _path_name(bs_model, filename):  # pylint:disable=unused-argument
+def _path_name(bs_model, _filename):
     """
```

Comment:
^Why is this even in the method signature?

Reply:
It's called by Django with those arguments to determine the upload path: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.FileField.upload_to